docs/README.md

```diff
@@ -35,7 +35,7 @@ To make this simple concept work, deck.gl handles a number of challenges:
 
 ## Ecosystem
 
-deck.gl is one of the main frameworks in the [vis.gl](https://vis.gl) framework suite.
+deck.gl is one of the main frameworks in the [vis.gl](http://vis.gl) framework suite.
```

Comment:
If vis.gl doesn't support https we should file a bug on that repo @jckr @balthazar

Reply:
Custom domains hosted on github pages do not support SSL

Reply:
> Custom domains hosted on github pages do not support SSL

Hmm, I thought it worked for our other websites but seems it doesn't. Pity...